WcaOnRails/config/locales/en.yml

```diff
@@ -762,7 +762,7 @@ en:
       faq: "Frequently Asked Questions"
       forum: "Forum"
       teams: "Teams, Committees, and Councils"
-      tutorial: "Competition Tutorial"
+      education: "Education resources"
```

Comment:
@moralsh @Ivan-Brigidano @AlbertoPdRF: would one of you mind providing me a suitable Spanish translation for that navigation entry please?

Reply:
`Recursos educativos` should be good enough.

Reply:
Awesome thanks :)